docker-sync.yml

```diff
@@ -0,0 +1,13 @@
+version: '2'
+
+options:
+  verbose: true
+
+syncs:
+  spryker-sync:
+    src: 'PATH_TO_SOURCE'
+    sync_excludes: ['EXCLUDE_1', '...', '.EXCLUDE_N']
```

Comment:
Is that correct?

Reply:
The docker-compose.yml and docker-sync.yml are starting points for beginners.

Reply:
Maybe the file extension .dist is better for this files.